cmd/csaf_checker/processor.go

```diff
@@ -441,6 +445,12 @@ func (p *processor) integrity(
 		}
 		p.checkTLS(u)
 
+		// Check if the filename is confirming.
```

Comment:
In my understanding _confirming_ is not the right word. It means the action of confirming something, not the item is confirmed itself. Maybe ""conforming"?

Reply:
But this does not affect the funcationality.